scripts/install-chef-suse.sh

```diff
@@ -1166,6 +1154,18 @@ for s in xinetd dhcpd apache2 ; do
     fi
 done
 
+# activate provisioner repos
+if ! is_ses; then
+    for i in SLES12-SP1-Pool SLES12-SP1-Updates SUSE-OpenStack-Cloud-6-Pool SUSE-OpenStack-Cloud-6-Updates SLES12-SP1-HA-Pool SLES12-SP1-HA-Updates; do
+      curl --digest -u crowbar:crowbar -X POST -d "platform=suse-12.1" -d "repo=$i" http://$IP/utils/repositories/activate
```

Comment:
hmm, would you use `json-read` for that? I would have to parse this string then:
```
attributes.crowbar.users={"crowbar"=>{"password"=>"crowbar"}}
```
right? or is there a more elegant way?